src/LiveComponent/doc/index.rst

```diff
@@ -1326,6 +1326,51 @@ The files will be available in a regular ``$request->files`` files bag::
     need to specify ``multiple`` attribute on HTML element and end ``name``
     with ``[]``.
 
+.. _downloads:
+
+Downloading files
+-----------------
+
+Currently, Live Components do not natively support returning file responses directly from a LiveAction. However, you can implement file downloads by redirecting to a route that handles the file response.
+
+Create a LiveAction that generates the URL for the file download and returns a `RedirectResponse`::
+
+    use Symfony\Component\HttpFoundation\RedirectResponse;
+    use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+    use Symfony\UX\LiveComponent\Attribute\LiveAction;
+
+    // ...
+
+    class MyDownloadButton
+    {
+    // ...
+
+        #[LiveAction]
+        public function initiateDownload(UrlGeneratorInterface $urlGenerator): RedirectResponse
+        {
+            $url = $urlGenerator->generate('app_file_download');
+            return new RedirectResponse($url);
+        }
```

Comment:
I'd say this is enough content here, just to illustrate. The documentation is already very long, let's focus on the valuable content (the paragraph before and the tip below)